cmd/cluster/transferowner.go

```diff
@@ -279,14 +342,30 @@ func verifyClusterPullSecret(clientset *kubernetes.Clientset, expectedPullSecret
 	if !ok {
 		return fmt.Errorf("pull secret data not found in the secret")
 	}
+	err = comparePullSecretAuths(pullSecret, expectedAuths)
+	if err != nil {
+		red.Printf("\nFound mis-matching auth values during compare. Please review:\n%s", err)
+		fmt.Printf("Would you like to continue?")
+		if !utils.ConfirmPrompt() {
+			return fmt.Errorf("operation aborted by the user")
+		}
+	} else {
+		//fmt.Printf("\nComparison shows subset of Auths from OCM AuthToken have matching tokens + emails in cluster pull-secret. PASS\n")
+		green.Println("\nComparison shows subset of Auths from OCM AuthToken have matching tokens + emails in cluster pull-secret. PASS")
+	}
 
-	fmt.Println("Actual Cluster Pull Secret:")
+	blue.Println("Actual Cluster Pull Secret:")
 	fmt.Println(string(pullSecretData))
```

Comment:
I know this wasn't introduced in this PR, but I'm a bit surprised that we just print the decoded pull-secret to stdout like this

Reply:
@nephomaniac - you may have the most comprehension around this subcommand at the moment, do you know if this is necessary for the user to complete an owner transfer/pull-secret update task?

Reply:
Mostly wondering if this is worth hiding behind a `-v` flag or something in a follow-up PR. That way we can still use the output if/when its needed, but it won't appear by default on screenshares or copy/pasted output

Reply:
I've left the option to dump the secret data to the terminal, prompting the user to optionally do so with a warning.  The newly added checks per auth section might provide enough user to the user executing this to make an informed decision about the status w/o the raw secret printed to the terminal.